acquire/gui/win32.py

```diff
@@ -0,0 +1,469 @@
+
+        wndclass = WNDCLASSW()
+        wndclass.style = CS_HREDRAW | CS_VREDRAW
+        wndclass.lpfnWndProc = WNDPROC(_winmessage)
+        wndclass.cbClsExtra = wndclass.cbWndExtra = 0
+        wndclass.hInstance = kernel32.GetModuleHandleW(None)
+        wndclass.hIcon = user32.LoadIconW(None, IDI_APPLICATION)
+        wndclass.hCursor = user32.LoadCursorW(None, IDC_ARROW)
+        wndclass.hbrBackground = gdi32.GetStockObject(WHITE_BRUSH)
+        wndclass.lpszMenuName = None
+        wndclass.lpszClassName = "AcquireGUI"
+        user32.RegisterClassW(byref(wndclass))
+        hwnd = user32.CreateWindowExW(
+            0,
+            wndclass.lpszClassName,
+            "Acquire",
+            WS_OVERLAPPEDWINDOW,
+            CW_USEDEFAULT,
+            CW_USEDEFAULT,
+            600,
+            400,
+            None,
+            None,
+            wndclass.hInstance,
+            None,
+        )
+
+        user32.ShowWindow(hwnd, SW_SHOWNORMAL)
+        user32.UpdateWindow(hwnd)
+        ole32.CoInitialize(None)
+        control_list = INITCOMMONCONTROLSEX()
+        control_list.dwSize = sizeof(INITCOMMONCONTROLSEX)
+        control_list.dwICC = 0x00000020
+        controls_loaded = comctl32.InitCommonControlsEx(byref(control_list))
+
+        if not controls_loaded:
+            raise GUIError("Unable to load GUI controls")
+
+        hFont = gdi32.CreateFontA(0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, b"Segoe UI")
+
+        if self.upload_available:
+            self.checkbox = user32.CreateWindowExW(
+                0, "Button", None, WS_CHILD | WS_VISIBLE | BS_AUTOCHECKBOX, 20, 250, 16, 16, hwnd, 0, 0, 0
+            )
+            self.upload_label = user32.CreateWindowExW(
+                0, "static", "upload", WS_CHILD | WS_VISIBLE | SS_LEFT, 50, 250, 100, 32, hwnd, 0, 0, 0
+            )
+            if hFont:
+                SendMessage(self.upload_label, WM_SETFONT, hFont, 1)
+            GUI.auto_upload = True
+            SendMessage(self.checkbox, BM_SETCHECK, 1, 0)
+
+        self.progress_bar = user32.CreateWindowExW(
+            0,
+            "msctls_progress32",
+            None,
+            WS_CHILD | WS_VISIBLE | WS_BORDER | PBS_SMOOTH,
+            20,
+            300,
+            550,
+            32,
+            hwnd,
+            0,
+            0,
+            0,
+        )
+
+        self.info = user32.CreateWindowExW(
+            0, "static", "Acquire output folder:", WS_CHILD | WS_VISIBLE, 20, 20, 200, 20, hwnd, 0, 0, 0
+        )
+        self.label = user32.CreateWindowExW(
+            0, "static", "No path selected...", WS_CHILD | WS_VISIBLE, 20, 40, 400, 25, hwnd, 0, 0, 0
+        )
+        self.choose_folder_button = user32.CreateWindowExW(
+            0, "Button", "choose folder", WS_CHILD | WS_VISIBLE | WS_BORDER | BS_FLAT, 450, 35, 100, 32, hwnd, 0, 0, 0
+        )
+        self.start_button = user32.CreateWindowExW(
+            0,
+            "Button",
+            "start",
```

Comment:
```suggestion
            "Start",
```
Again I'm not sure if and where these strings are visible? But if they are user visible they should be capitalised.